### ISSUE TITLE
Static caveats without text display in results

### ISSUE THREAD

**Issue by bsnouffer** [shcaba/FishPath-updates#230]
A couple options have a static caveats without descriptions which displays like this:

![image](https://user-images.githubusercontent.com/11687620/53216902-f1741400-360a-11e9-9e37-4a32cf60765d.png)

It occurs for these options:
Catch curve analysis (assessment)
Production model (assessment)
permanent no-take zones (control rules)

This appears to be a software bug arising from a static caveat being added, then not having text written. Caveats without text should be disabled, but these still show up in the results. 

For the short term, I assume that these are just input errors and should just be deleted from the admin interface and keep an eye out when inputting static caveats until this bug is fixed. 

Here is what the admin interface looks like:

Catch curve analysis 
![image](https://user-images.githubusercontent.com/11687620/53216866-d3a6af00-360a-11e9-99e4-03abb08d1b8c.png)

Permanent no take zone
![image](https://user-images.githubusercontent.com/11687620/53216884-ddc8ad80-360a-11e9-8513-924146f2a34b.png)
Showed both as it doesn't appear to matter if there is another caveat in the list for this bug to appear.

**Comment by bsnouffer** [comment]
I used the caveat export spreadsheet to determine the options it appeared in. It doesn't appear to be an issue for normal caveats.

**Comment by dtdougherty** [comment]
I removed the blank under permanent no-take zones and production model. However, when I tried to remove the blank caveat under catch curves, it kept deleting the one above it instead, so I left it alone for now.

**Comment by bsnouffer** [comment]
I managed to delete the last blank under catch curve. For some reason it had the same id as the caveat above which is why clinking the delete button for the blank deleted the one above it. I don't think there is much we can do about it now to determine how that happened, but noting it here in case we come across another instance of duplicate IDs.

I have also tested a software update that will not save any static caveat with no text to prevent this from happening in the future. I will close this issue once the fix is uploaded.
